<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_webcamfeed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>561</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_webcam_feed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Web camp Feed</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_processed_webcamfeed">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>561</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>10</y>
     <width>1141</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_start">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start Webcamp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_stop">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Stop/Pause</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_track">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Track Color</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>560</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>H_Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sldr_h_max">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>S_Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sldr_s_max">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>V_Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sldr_v_max">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbl_color_1_value">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>561</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Value: </string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>H_Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sldr_h_min">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>S_Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sldr_s_min">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>V_Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sldr_v_min">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>153</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QCheckBox" name="ckb_color_1">
      <property name="text">
       <string>Color 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_set_color_1">
      <property name="text">
       <string>Set color 1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>lbl_webcamfeed</zorder>
  <zorder>lbl_webcam_feed</zorder>
  <zorder>lbl_processed_webcamfeed</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>sldr_h_min</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>sldr_s_min</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>label_10</zorder>
  <zorder>btn_set_color_1</zorder>
  <zorder>lbl_color_1_value</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
